### ISSUE TITLE
Add a README

### ISSUE THREAD

**Issue by mc0e** [ayufan/dokku-alt-phpmyadmin#2]
I realise that the purpose of this is mostly explained by external pages, but still...

**Comment by asimzeeshan** [comment]
Yes, I also need a README.md file to help me understand how to get this installed. //noob question

**Comment by robinwo** [comment]
How can I get this to work?
1. Switched so sudo
2. Cloned this repo to /var/lib/dokku-alt/plugins/dokku-alt-phpmyadmin/
3. Modified blowfish in config.inc.php
4. Run "docker build ."
5. Run "dokku plugins-install"

Now when I call "dokku plugins", this phpmyadmin plugin does not show up. What am I doing wrong?

**Comment by mc0e** [comment]
This is not a plugin, it's a container build spec.  It'll go something like this:

```
git clone https://github.com/ayufan/dokku-alt-phpmyadmin.git
cd dokku-alt-phpmyadmin
git remote add dokku@<yourserver.example.com>:phpmyadmin
git push
```

phpmyadmin will need to know where to find the mysql server.
